client/components/domains/register-domain-step/index.jsx

```diff
@@ -524,7 +524,8 @@ const RegisterDomainStep = React.createClass( {
 
 	showValidationErrorMessage: function( domain, error ) {
 		let message,
-			severity = 'error';
+			severity = 'error',
+			tldIndex;
```

Comment:
Why use tldIndex? You can declare the `tld` variable here and never use `tldIndex`. It shoud also be a const.